README.md

```diff
@@ -61,13 +62,17 @@ This command will download the script from the specified URL and execute it with
 - --inventory <PATH>: Specify the path to the inventory file (required for --remote).
 - --ssh-key <SSH_KEY_FILE>: Specify the path to the SSH key file (required for --remote).
 - --user <USER>: Specify the user for running Ansible (required for --local).
+- --tags <TAGS>: Specify tags for Ansible playbook tasks (comma-separated, e.g., 'devops,shell,desktop').
+
+### Ansible Tags
+The tasks for installing `VLC` and `Telegram` have `desktop` tag, all other tools have `devops` tag and all shell utilities have `shell` tag. You can find their details at the [Tools Installed 🧰](#tools-installed-) section.
```

Comment:
It sounds ambiguous for me. I recommend categorize and explain them by category. Like:
Tags:
- Desktop:
  - VLC
  -   Telegram
  - ...
- DevOps
  - Docker
  - ...
- Shell
  - Byobu
  - ...